app/jobs/quickbooks_update_job.rb

```diff
@@ -34,14 +35,10 @@ def perform(_job_params)
     end
 
     # TODO: This is duplicated in Updater#update and needs to be DRYed up.
-    # We record last_updated_at as the time the update *finishes* because last_updated_at
-    # is used to avoid runs that immediately follow each other as in the case of a transaction creation
-    # followed by a transaction listing. If we record the time the update *started* and the update
-    # takes some time, we would need to increase the MIN_TIME_BETWEEN_UPDATES value and that might
-    # make it frustrating for users who want to deliberately re-run the updater.
-    # The other function of last_updated_at is to check if a full sync needs to be run,
-    # but that condition is measured in days, not seconds, so this small a difference shouldn't matter.
-    updater.qb_connection.update_attribute(:last_updated_at, Time.current)
+    # We record last_updated_at as the time this update started. The user-prompted ways
+    # the update is started are used by only admins and rarely.
```

Comment:
I think I follow your logic here! But this comment is written seemingly as a response to the old one, and without the old one as context, the second sentence is kind of hard to grasp. So perhaps some more detail here would be useful, perhaps just saying that it's important to use update start time so as not to miss txns. Simple as that really, isn't it?

Separately: the old comment mentions `MIN_TIME_BETWEEN_UPDATES`. Do we need to increase that? Or perhaps more correctly, do we need some way of preventing multiple updates running at one time and causing havoc? That sounds bad and I'm not convinced we are dealing with it properly right now, especially if we are changing the below... 

Like what if someone clicks the sync button before the previous sync finishes?